clouddriver-kubernetes/src/main/groovy/com/netflix/spinnaker/clouddriver/kubernetes/v2/op/handler/KubernetesClusterRoleBindingHandler.java

```diff
@@ -28,7 +28,7 @@
 import static com.netflix.spinnaker.clouddriver.kubernetes.v2.op.handler.KubernetesHandler.DeployPriority.ROLE_BINDING_PRIORITY;
 
 @Component
-public class KubernetesClusterRoleBindingHandler extends KubernetesHandler implements CanDelete {
+public class KubernetesClusterRoleBindingHandler extends KubernetesHandler implements CanDelete, CanPatch {
```

Comment:
Any reason not to add `CanPatch` to `KubernetesHandler`? Are there any resources that don't support it?

Reply:
done!